### PR TITLE
[SIM_FLUTTER-82] [BE] Apply linters to the repository

### PR DESCRIPTION
**Pull request by jescabillas**
**Issue Link**
- [BackLog](https://framgiaph.backlog.com/view/SIM_FLUTTER-82)

**Definition of done**
- [x] [BE] Apply linters to the repository

**Note**

To run the linters in laravel
- Make sure you're in the backend directory then run `./vendor/bin/pint` in the terminal

Or
1. Add keyboard shortcut in your vscode `file -> preferences -> keyboard Shortcuts`
2. Click on the file icon to open the json file of the keyboard shortcuts
3. Once open, add the following code to the file
````
[
  {
    "key": "ctrl+shift+l",
    "command": "workbench.action.tasks.runTask",
    "args": "Pint Formatter"
  }
]
````
4. Now all you have to do is hit `Ctrl + Shift + l` every time you want to run the linter

**Attachment**
image
image

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+            "presentation": {
+                "reveal": "silent"
+            },
+            "group": {
+                "kind": "build",
+                "isDefault": true
+            }
+        }
+    ]
+}
```
Comment (karlolee50) [inline .vscode/tasks.json]: add extra line here

File: backend/composer.lock [hunk 2]
```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "c61ff82cbf0142a401a48a8161e1595a",
+    "content-hash": "82d11cab70c32144370c683cf476478e",
```
Comment (karlolee50) [inline backend/composer.lock]: do we need to push composer.lock? if not needed, can you update the .gitignore file?

**Review by karlolee50 — CHANGES_REQUESTED** [review]

File: .vscode/tasks.json (hunk 1 above)
Comment (jescabillas) [reply .vscode/tasks.json]: Added extra line

File: backend/composer.lock (hunk 2 above)
Comment (jescabillas) [reply backend/composer.lock]: Added the composer.lock in .gitignore and reset the commits to exclude the changes in composer.lock

**Review by karlolee50 — DISMISSED** [review]

**Review by Dbulawan13 — APPROVED** [review]
Approve PR #5 

Working:
- Linters is working properly
- Able to check if codes are pass or fail




Testing Hours: 3
Comment: 0
Iteration: 0

**Pull request merged**